data/scopes/yealink-T29G.ini

```
@@ -6,3 +6,33 @@ displayName = "Yealink T29G"
 [data]
 provisioning_url_filename = 
 tmpl_phone = 'yealink.tmpl'
+cap_linekey_count = 27
+cap_linekey_type_blacklist = ""
+cap_softkey_count = 4
+cap_softkey_type_blacklist = ""
+cap_expkey_count = 38
+cap_expkey_type_blacklist = 
+cap_ringtone_count = 
+cap_ringtone_blacklist = 
+cap_expmodule_count = 3
+
+softkey_type_1 = "history"
+softkey_value_1 = ""
+softkey_label_1 = "History"
```

Remove localized labels i.e.:

```suggestion
softkey_label_1 =
```